wiki-service/src/main/java/org/exoplatform/wiki/service/impl/WikiServiceImpl.java

```
@@ -416,7 +416,11 @@ public String getWikiNameById(String wikiId) throws WikiException {
 
   @Override
   public Wiki createWiki(String wikiType, String owner) throws WikiException {
-    Wiki wiki = new Wiki(wikiType, owner);
+    Wiki wiki = getWikiByTypeAndOwner(wikiType, owner);
+    if (wiki != null) {
+      return wiki;

```

We should throw an exception if we try to create a wiki which already exists IMO.

---

done